templates/taxi/manufacturer_list.html

```diff
@@ -10,6 +10,8 @@ <h1>Manufacturer List
           <th>ID</th>
           <th>Name</th>
           <th>Country</th>
+          <th>Update</th>
+          <th>Delete</th>
```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Delete' one. The 'Cancel' button should lead to the previous page the user was on.